### DISCUSSION TITLE
Search via TraceQL for span links

### DISCUSSION THREAD

**Discussion by ovidiubuligan** [grafana/tempo#3022] [Q&A] [answered]
Currently I don't see it possible at all . The only alternative is to store the linked traceId as an attribute wich is not optimal ?

Any thoughts on this ?

**Comment by joe-elliott** [comment, marked as answer]
We are working on a new version of Parquet and defining the syntax to query links now. It will likely be a bit before it makes it into Tempo, but it's in the works!

cc @stoewer

**Comment by ovidiubuligan** [comment]
If someone comes from google this is the documentation link https://grafana.com/docs/tempo/latest/traceql/#selecting-spans 

TLDR:
```promql
{ link:traceID = "1234" }
```
